src/Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -202,7 +199,7 @@ public function getValueFromFieldDescription($object, FieldDescriptionInterface
     /**
      * @return string
      */
-    protected function getClass(FormBuilderInterface $formBuilder)
+    private function getClass(FormBuilderInterface $formBuilder)
```

Comment:
`@return` docblock can be removed:
```suggestion
    private function getClass(FormBuilderInterface $formBuilder): string
```

Reply:
Fixed